csharp/ql/src/Security Features/CWE-114/AssemblyPathInjection.ql

```diff
@@ -22,7 +22,7 @@ import AssemblyPathInjection::PathGraph
  */
 module AssemblyPathInjectionConfig implements DataFlow::ConfigSig {
   predicate isSource(DataFlow::Node source) {
-    source instanceof RemoteFlowSource or
+    source instanceof ThreatModelFlowSource or
     source.asExpr() = any(MainMethod main).getParameter(0).getAnAccess()
```

Comment:
This is basically a local source (we shouldn't do anything in this PR), but if main parameters are modelled generally and being "configurable" via threat models then we should consider to remove the hardcoding of main parameters here.